gutenberg.php

```diff
@@ -517,13 +485,14 @@ function gutenberg_replace_default_add_new_button() {
 			}
 
 			var url = button.href;
-			var newUrl = url.replace( /[&\?]?classic-editor/, '' );
+			var urlHasParams = ( -1 !== url.indexOf( '?' ) ); 
```

Comment:
There was a single trailing whitespace here, which is apparently not captured by PHPCS (likely because it's in a JS code block?). I'd encourage you to install an EditorConfig integration for your preferred editor, which will automatically strip this on a per-project basis (configured already for Gutenberg):

http://editorconfig.org/#download

Fixed in 3e18927a64cec0027a8ee7f25aec2cbf804c2b8d